python/tests/test_aac/in_out/test_paths.py

```diff
@@ -1,12 +1,10 @@
 import os
-# from nose2.tools import params
+from parameterized import parameterized
 from unittest import TestCase
-
 from aac.in_out.paths import sanitize_filesystem_path
```

Comment:
I wonder if this is somewhere within the logic we should look at bubbling some file handle errors up from if it is a true file location issue.

Thoughts?

Reply:
That looks like a good place.